test/EFCore.Specification.Tests/GraphUpdates/GraphUpdatesTestBaseMiscellaneous.cs

```diff
@@ -1570,4 +1569,132 @@ public virtual void Can_add_multiple_dependents_when_multiple_possible_principal
                         + hiddenAreaTask.Choices.Count(e => e.Id == taskChoice.Id));
                 }
             });
+
+    [ConditionalFact]
+    public void Cab_attach_full_required_graph_of_duplicates()
```

Comment:
```suggestion
    public void Can_attach_full_required_graph_of_duplicates()
```
Also in the other tests